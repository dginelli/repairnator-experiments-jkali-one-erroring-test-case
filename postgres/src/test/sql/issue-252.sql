create table issue252 (
  id serial8,
  data bytea
);

insert into issue252 (data) values ('123456');
create index idxissue252 on issue252 using zombodb (zdb('issue252', ctid), zdb(issue252)) with (url='localhost:9200/');

select zdb_get_index_mapping('issue252')->'mappings'->'data'->'properties'->'data'->>'type' as es_type;
drop table issue252 cascade;