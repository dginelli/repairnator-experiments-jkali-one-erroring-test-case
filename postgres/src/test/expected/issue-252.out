create table issue252 (
  id serial8,
  data bytea
);
insert into issue252 (data) values ('123456');
create index idxissue252 on issue252 using zombodb (zdb('issue252', ctid), zdb(issue252)) with (url='localhost:9200/');
select zdb_get_index_mapping('issue252')->'mappings'->'data'->'properties'->'data'->>'type' as es_type;
 es_type 
---------
 keyword
(1 row)

drop table issue252 cascade;
